README.md

```diff
@@ -21,4 +25,18 @@ Currently, a test packet is created and placed into the compute node's TX queue.
 
 When the memory node receives a "read" packet for data, it retrieves the value and sends a packet back to the compute node with the data value.  These packets can be sent concurrently.
 
-Next steps are to add logic to handle sending to different memory nodes/compute nodes through a single switch (different ports).
\ No newline at end of file
+Next steps are to add logic to handle sending to different memory nodes/compute nodes through a single switch (different ports).
+
+## Packet input/log file format
+
+The input/log file are csv (comma separated values) files with the following columns:
+
+**Time, Packet ID, Flag, src, dst, Address, Data**
```

Comment:
I think we should have the input log say if its a read or write instruction. I don't see any provisioning for actually modifying the data itself in the input file.

Reply:
Should be included in the "Flag" field.  See "packet.h" NORMAL=0, READ=1, WRITE=2, ERROR=3.

Reply:
oops didn't see that